README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
```

Comment:
The measurement of a TEE, guest boot flow, or OS is per CC VM  <-- I think it is the guest boot flow you are measuring.  Is it a single measurement or multiple measurements? The way it is written it sounds like a single measurement is done across those three things.  If it is more than one measurement then it would be  "measurements"  and "is" would become "are".

Reply:
It's a single measurement for TEE + guest boot + OS because it's about a single TD.